schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+            "type": "string",
+            "format": "date-time"
+          },
+          "license": {
+            "title": "License",
+            "description": "The license associated with the content.",
+            "oneOf": [
+              {"$ref": "#/definitions/active_licenses"},
+              {"$ref": "#/definitions/inactive_licenses"}
+            ]
+          },
+          "titles": {
+            "title": "Titles",
+            "description": "The English and non-English versions of the title.",
+            "$ref": "#/definitions/titles"
+          },
+          "abstracts": {
+            "$ref": "#/definitions/abstracts"
+          },
+          "legacy_authors": {
+            "title": "Legacy authors",
+            "description": "The legacy authors field, which is an unstructured string.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "J. Doe (Cornell University), M. Smith (Stanford University), F. Bar"
+            ]
+          },
+          "authors": {
+            "title": "Authors",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/person"
+            },
+            "minItems": 1
+          },
+          "collaborations": {
+            "title": "Collaborations",
+            "description": "Collaborations associated with this e-print. Should not include 'Collaboration' suffix.",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/collaboration"
+            },
+            "examples": [
+              "ATLAS",
+              "BABAR",
+              "L3",
+              "ZEUS"
+            ],
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "comments": {
+            "title": "Comments",
+            "description": "Comments from authors or arXiv administrators.",
+            "type": "object",
+            "properties": {
+              "author_comments": {
+                "Title": "Author comments",
+                "description": "Comments from authors.",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "administrator_comments": {
+                "title": "Administrator comments",
+                "description": "Comments from by arXiv administrators",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "legacy_comments": {
+                "description": "Unstructured comments from authors and/or arXiv administrators.",
+                "$ref": "#/definitions/comment"
+              }
+            }
+          },
+          "journal_ref": {
```

Comment:
why one journal ref, but multiple publisher DOIs? aren't those expected to map one-to-one?

Reply:
This follows the legacy schema in that those fields are siblings and have no explicit dependency on one another. It makes sense to have them map one-to-one, but would need to analyze how well that would work with the legacy values.